tools/topology/sof-hda-generic-idisp.m4

```diff
@@ -0,0 +1,97 @@
+#
+# Topology for generic Apollolake UP^2 with pcm512x codec and HDMI.
```

Comment:
@plbossart only minor but this comment needs to be amended.

Reply:
ok